app/views/expenses/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<p style="color: green"><%= notice %></p>
+
+<div class='expenditure-card'>
+ <div class='expense-heading'>
+    <h1>EXPENDITURE</h1>
+   </div>
+
+  <div class='description'>
+     <%= image_tag @expense.icon, alt: @expense.name %>
+     <p><%= @expense.name %></p>
+     
+  </div>
+
+  <% totalAmount = 0 %>
+  <% @expense.expenditures.each do |expenditure| %>
+
+   <div class='expenditure-detail'>
+     <div class='expenditure-id'>
+       <div>
+         <p> Expenditure No:<%= expenditure.id%></p>
+         <p>Amount spent:$<%= expenditure.amount%></p>
+        </div>
+        <%= button_to "Delete", expenditure, method: :delete %>
+      </div>
+      <p><%= expenditure.created_at.strftime('%d %B %Y') %> at <%= expenditure.created_at.strftime('%I:%M %p') %></p>
+    </div>
+    <% totalAmount += expenditure.amount %>
+  <% end %>
+
+  <div class='tot-payment'>
+     <p>TOTAL PAYMENT:</p>
+     <h2>$<%= totalAmount%></h2>
+   </div>
+</div>
+
+
+<%= link_to "Back to expenses", expenses_path %>
```

Comment:
- [ ] This link should be inside the header to be similar to the original design, please correct its position and use icons 👇🏽

| Original design | Your design |
|---|---|
| ![image](https://github.com/prius2055/rails-budget-app/assets/58642949/bc85f162-0443-4b8e-9a92-56153103412a) | ![image](https://github.com/prius2055/rails-budget-app/assets/58642949/2115ed44-4e3e-4650-be78-99fed94c0799) |

Reply:
Done. Thanks for this

Reply:
Fixed. Thank you